src/System.Linq.Expressions/src/System/Linq/Expressions/Compiler/LambdaCompiler.Expressions.cs

```diff
@@ -862,24 +860,6 @@ private void EmitMemberGet(MemberInfo member, Type objectType)
             }
         }
 
-        private static bool TryGetRawConstantValue(FieldInfo fi, out object value)
-        {
-            // TODO: It looks like GetRawConstantValue is not available at the moment, use it when it comes back.
-            //value = fi.GetRawConstantValue();
-            //return true;
-
-            try
-            {
-                value = fi.GetValue(obj: null);
-                return true;
-            }
-            catch
-            {
-                value = null;
```

Comment:
Had this catch block just been belt-and-braces? It isn't hit by any tests, and I can't see how it would be hit if `fi.IsLiteral` but the logic of the removed code seems to be worried about that possibility